frontend/src/libs/packages/store/middlewares/handle-unauthorized/handle-unauthorized.middleware.ts

```diff
@@ -0,0 +1,27 @@
+import { type AnyAction, type Middleware } from '@reduxjs/toolkit';
+import { isRejected } from '@reduxjs/toolkit';
+
+import { ServerErrorType } from '#libs/enums/enums.js';
+import { type AppDispatch } from '#libs/packages/store/libs/types/types.js';
+import { actions as authActions } from '#slices/auth/auth.js';
+
+const handleUnauthorized: Middleware<unknown, unknown, AppDispatch> = ({
+  dispatch,
+}) => {
+  return (next) => {
+    return (action: AnyAction) => {
+      if (isRejected(action)) {
```

Comment:
I believe that isRejected(action) is useless bcos action.error.name === ServerErrorType.AUTHORIZATION will kill two birds with one stone the defendant

Reply:
@v0ldemar01  without isRejected just on the page load I get this
![isRejected](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/104642736/e49967b9-5572-4e71-8059-24d3271336a5)

And I am basically stuck on the sign in page after it

Reply:
@OlesiaTr but action.error.name === ServerErrorType.AUTHORIZATION is only for 401 case and it includes rejected action